controllers/openstackconfiggenerator_controller.go

```diff
@@ -521,11 +521,22 @@ func (r *OpenStackConfigGeneratorReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, err
 	}
 
+	// obtain the cltplaneExports from Heat
+	exports, err := openstackconfiggenerator.CtlplaneExports("heat-"+instance.Name, r.Log)
```

Comment:
FIXME: Need to run this only conditionally *if* the stack is for Ctlplanes (has Controllers)